app/main.py

```
@@ -1,6 +1,24 @@
+import os
+
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    API_KEY = os.getenv("API_KEY", "No API key provided")
```

Consider handling the case where the API key is not provided more gracefully. Currently, it defaults to 'No API key provided', which will not work with the API. You might want to raise an exception or log a warning if the API key is missing.